core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-contentlets/edit-ema-palette-contentlets.component.scss

```diff
@@ -0,0 +1,87 @@
+@use "variables" as *;
+
+:host {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+
+    ::ng-deep {
+        .p-paginator-icon {
+            color: $color-palette-primary;
+        }
+    }
+}
+
+.content-search {
+    display: flex;
+    align-items: center;
+    gap: $spacing-0;
+    top: 0;
+    padding: $spacing-3 $spacing-3;
+    position: sticky;
+    z-index: 99;
+    background: white;
+
+    .p-input-icon-right {
+        width: 100%;
+        input {
+            width: 100%;
+        }
+    }
+}
+
+.contentlets {
+    display: flex;
+    flex-direction: column;
+    padding: 0 $spacing-3;
+    height: 100%;
```

Comment:
`flex-grow`?

Reply:
Height is set here for the overflow, flex-grow is not needed i think